modules/git/sha1_nogogit.go

```diff
@@ -1,61 +0,0 @@
-// Copyright 2015 The Gogs Authors. All rights reserved.
-// Copyright 2019 The Gitea Authors. All rights reserved.
-// SPDX-License-Identifier: MIT
-
-//go:build !gogit
-
-package git
-
-import (
-	"crypto/sha1"
-	"encoding/hex"
-	"hash"
-	"strconv"
-)
-
-// SHA1 a git commit name
-type SHA1 [20]byte
```

Comment:
I would like to keep this definition so that when we need this type we can use it as before.

Reply:
Where would you like to use it? I'm not sure it's really beneficial to keep hashes as distinct types if SHA256 and SHA1 are to be interchangeable.

Reply:
I think I understand. I'll see what I can do about it to keep this type as it was.

Reply:
Like I did in my PR. You don't need to define
```go
type Sha1Hash struct {
	val [20]byte
}
```
but add some methods for `SHA1` here.